### DISCUSSION TITLE
My relay output is inverted

### DISCUSSION THREAD

**Discussion by Modbusinternet** [forkineye/ESPixelStick#515] [General]
Hello there, I try the software with relays, so i could make then ON e OFF, the issue is that my signal is inverted when i send a ON signal they go OFF, is any chance of helping me? Maybe i am using it wrong. 
Thanks for your patience
Good afternoon

**Comment by MartinMueller2003** [comment]
There is an inverted checkbox. The default is "output is active when low" aka the inverted box is checked. If your relays use "Output is active when high" then uncheck the box.
